### PR TITLE
Add predicate support to predicated alt saver

### PR DESCRIPTION
**Pull request by sarthak-marwaha**

### CODE REVIEW

File: src/foam/u2/view/RichChoiceView.js [hunk 1]
```diff
@@ -529,7 +529,7 @@ foam.CLASS({
                 this.addClass(self.myClass('setAbove'))
                   .start().hide(!! section.hideIfEmpty && resp[index].value <= 0 || ! section.heading)
                     .addClass(self.myClass('heading'))
-                    .translate(section.heading, section.heading)
+                    .add(section.heading$)
```
Comment (kgrgreer) [inline src/foam/u2/view/RichChoiceView.js]: Translation is no longer needed?

Comment (kgrgreer) [reply src/foam/u2/view/RichChoiceView.js]: Are section headings dynamic?

Comment (kgrgreer) [reply src/foam/u2/view/RichChoiceView.js]: I see the dynamic header from Mobile wallet selector #16723.

Comment (sarthak-marwaha) [reply src/foam/u2/view/RichChoiceView.js]: Added slot support to translate instead, I see the problem now, forgot that strings can also be translated, not just messages

**Pull request merged**